internal/octant/workload.go

```diff
@@ -639,3 +645,13 @@ func CreateWorkloadSummary(workload *Workload, summarySize component.DonutChartS
 		MetricsEnabled: true,
 	}, nil
 }
+
+func ExternalMetricsUnavailable(e error) bool {
```

Comment:
Instead of parsing out strings from the error, we can try to catch the error in https://github.com/vmware-tanzu/octant/blob/master/internal/octant/pod_metrics_loader.go#L158 by using `discovery.IsGroupDiscoveryFailedError` to identify instead since `ExternalMetricsUnavailable` and `IsPodMetricsNotSupported` are really the same here.